### PR TITLE
[IMP] l10n_tr: Make account names and reports translatable

### PR DESCRIPTION
**Pull request by poma-odoo**
Keeping the account and account group names in English would make support easier.
Some minor fixes and updates to the chart of account has also been made. 
Translations have been matched to the Sworn Accountants' Organization [publication] whenever possible.

Task#: [2818046]
Task#: [2811138]

[publication]: https://archive.ismmmo.org.tr/yayinlar/ingilizce_yayinlar/TheUniformChartOfAccounts.doc
[2818046]: https://www.odoo.com/web#id=2818046&model=project.task
[2811138]: https://www.odoo.com/web#id=2811138&model=project.task

### CODE REVIEW

**Comment by mart-e** [comment]
@poma-odoo your convertion is in master but #88025 is for 15.0

**Comment by poma-odoo** [comment]
> @poma-odoo your convertion is in master but #88025 is for 15.0

Yes, this is why I erased my comment, sorry for the email though, I couldn't wipe it :)

**Comment by lk-eska** [comment]
- 7a is only used by very small companies and can be ignored in my opinion
- A tax is defined as tax groups 
- I see no benefit on multi language accounting chart unless it is going to be standard in odoo

**Comment by poma-odoo** [comment]
> * 7a is only used by very small companies and can be ignored in my opinion
- 7A/7B is selectable and we can have one of them as default, if it is not necessary I can happily remove it.
> * A tax is defined as tax groups
- I didn't get it, are you talking about a specific tax (which one?) or all taxes?
> * I see no benefit on multi language accounting chart unless it is going to be standard in odoo
- this is standard procedure now so the technical support team can see what is going on without having to know the client's language

**Comment by lk-eska** [comment]
> > * 7a is only used by very small companies and can be ignored in my opinion
> 
>     * 7A/7B is selectable and we can have one of them as default, if it is not necessary I can happily remove it.

I would suggest not to have this option

> > * A tax is defined as tax groups
> 
>     * I didn't get it, are you talking about a specific tax (which one?) or all taxes?

why is tax_group_kdv_18 a tax group? it is just a tax with the rate of 18%?

**Comment by poma-odoo** [comment]
> why is tax_group_kdv_18 a tax group? it is just a tax with the rate of 18%?

Chat groups are automatically displayed below invoices, this is the sole purpose of tax groups. AFAIK we have to report kdv 18%, kdv 8% and kdv1% separately below the invoice.

I think you might have group type tax into mind (`tax.amount_type='group'`)which is a collection of different taxes

File: addons/l10n_tr/__manifest__.py [hunk 1]
```diff
@@ -13,10 +13,10 @@
     * Sihirbaz sizden hesap planı şablonu, planın kurulacağı şirket, banka hesap
       bilgileriniz, ilgili para birimi gibi bilgiler isteyecek.
     """,
-    'author': 'Ahmet Altınışık, Can Tecim',
-    'maintainer':'https://launchpad.net/~openerp-turkey, http://www.cantecim.com',
+    'author': 'Odoo',
     'depends': [
         'account',
+        'l10n_multilang',
```
Comment (william-andre) [inline addons/l10n_tr/__manifest__.py:19]: Where is the call to `load_translations` in the post init hook?

**Review by william-andre — COMMENTED** [review]
There is no enterprise PR, are we not translating the reports?

File: addons/l10n_tr/__manifest__.py (hunk 1 above)
Comment (poma-odoo) [reply addons/l10n_tr/__manifest__.py:19]: This boy is almost a year old, it was using the old way of `i18n_extra`, it is fixed now.

**Comment by poma-odoo** [comment]
> There is no enterprise PR, are we not translating the reports?

https://github.com/odoo/enterprise/pull/25979 it has a separate PR, it needs updating too.

File: addons/l10n_tr/__manifest__.py (hunk 1 above)
Comment (william-andre) [reply addons/l10n_tr/__manifest__.py:19]: `l10n_extra` doesn't change anything but ok...

**Comment by william-andre** [comment]
> it has a separate PR, it needs updating too

But why?

**Comment by poma-odoo** [comment]
> > it has a separate PR, it needs updating too
> 
> But why?

different times, different measures, but I'll move it here on Monday it makes sense

File: addons/l10n_tr/__manifest__.py (hunk 1 above)
Comment (smetl) [reply addons/l10n_tr/__manifest__.py:19]: If you add a dependency to `l10n_multilang`, you can remove the dependency to `account`

Comment (poma-odoo) [reply addons/l10n_tr/__manifest__.py:18]: I think it is good for information, and it is not changing anything, but done!

File: addons/l10n_tr/i18n/tr.po [hunk 2]
```diff
@@ -0,0 +1,1499 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_tr
+#: model:account.account.template,name:l10n_tr.tr278
+msgid "Accumulated Depletion (-)"
+msgstr "BİRİKMİŞ TÜKENME PAYLARI (-)"
```
Comment (william-andre) [inline addons/l10n_tr/i18n/tr.po:21]: IS IT EXPECTED THAT EVERYTHING IS IN ALL CAPS OR DID YOU JUST WRITE THIS ON CAPSLOKO FRIDAY?

**Comment by poma-odoo** [comment]
Turkish government is always in Capsoloko state, they even write the last
names in capitals, and the titles are also in capital letters. Any report
in any government samples and other accounting apps are like this
Pouya Malekinejad
Research and Development

[image: Odoo logo] <https://www.odoo.com>
Chaussée de Namur, 40
1367 Grand-Rosière (Belgium)

**Review by william-andre — APPROVED** [review]
@robodoo r+ squash

**Comment by william-andre** [comment]
@robodoo r+

**Pull request closed without merging**